src/Microsoft.AspNetCore.Cors/Infrastructure/CorsMiddleware.cs

```diff
@@ -40,10 +44,61 @@ public class CorsMiddleware
         /// <param name="corsService">An instance of <see cref="ICorsService"/>.</param>
         /// <param name="policyProvider">A policy provider which can get an <see cref="CorsPolicy"/>.</param>
         /// <param name="policyName">An optional name of the policy to be fetched.</param>
+        [Obsolete("This constructor has been replaced with an equivalent constructor which requires an ILoggerFactory")]
         public CorsMiddleware(
             RequestDelegate next,
             ICorsService corsService,
             ICorsPolicyProvider policyProvider,
+            string policyName
+            )
+            : this(next, corsService, policyProvider, NullLoggerFactory.Instance, policyName)
+        {
+        }
+
+        /// <summary>
+        /// Instantiates a new <see cref="CorsMiddleware"/>.
+        /// </summary>
+        /// <param name="next">The next middleware in the pipeline.</param>
+        /// <param name="corsService">An instance of <see cref="ICorsService"/>.</param>
+        /// <param name="policy">An instance of the <see cref="CorsPolicy"/> which can be applied.</param>
+        [Obsolete("This constructor has been replaced with an equivalent constructor which requires an ILoggerFactory")]
```

Comment:
Remove this constructor?

Reply:
GitHub is doing a bad job doing a diff. There were 3 original ctos - https://github.com/aspnet/CORS/blob/dev/src/Microsoft.AspNetCore.Cors/Infrastructure/CorsMiddleware.cs - that now have an overload each.